### DISCUSSION TITLE
10 degree sector taken out of slider starts visual glitch.

### DISCUSSION THREAD

**Discussion by Kn4ughty** [ppy/osu#29645] [Q&A] [answered]
### Type

Cosmetic

### Bug description

There is a 10 degree sector taken out of the starting circle imprint of a slider, the slice only appears on certain resolutions (happened at 1080p), and only [this map](https://osu.ppy.sh/beatmapsets/72324#osu/212095) at 2:39. Did not appear elsewhere in the song.

I am on sway arch linux, with an AMD 7600xt gpu.

Replay available here.
[Knaughty playing Sim Gretina - Raise This Barn (Remix) (ztrot) [TVO_s Hard] (2024-08-21_20-06).osr.zip](https://github.com/user-attachments/files/16690145/Knaughty.playing.Sim.Gretina.-.Raise.This.Barn.Remix.ztrot.TVO_s.Hard.2024-08-21_20-06.osr.zip)

### Screenshots or videos

![image](https://github.com/user-attachments/assets/be9d26a5-51e1-4a74-bdff-81be04939f23)

### Version

2024.817.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16690156/compressed-logs.zip)

**Comment by bdach** [comment]
Cannot reproduce, and I've messed with window size a fair amount.

Wouldn't discount possibility of driver bug.

**Comment by smoogipoo** [comment]
Also cannot reproduce, almost definitely a driver bug (likely linux related). I suggest updating Mesa/drivers, and if it still doesn't fix it you're going to have to go in manually via something like Renderdoc to see what's going on there.

**Comment by Kn4ughty** [comment]
To cut a long story short my drivers should be up to date. I switched to gnome x11 and captured frame of the error in a debug build.

Here it is.
[dotnet_2024.08.21_23.23.35_frame43855.zip](https://github.com/user-attachments/files/16700893/dotnet_2024.08.21_23.23.35_frame43855.zip)

**Comment by smoogipoo** [comment]
Unfortunately that doesn't help because it's rendered to a framebuffer :(

Does changing to the vulkan renderer in the in-game options help? I don't recommend using it, but it would be interesting to know at least. If it does, there is a future where we fix up the Vulkan renderer and will be able to recommend it.

**Reply by cloudrac3r** [reply, marked as answer]
Switching to Vulkan appears to fix it

**Comment by cloudrac3r** [comment]
I can reproduce on the linked map at 1920x1080 100% scaling with Argon skin, as well as on [this one](https://github.com/ppy/osu/issues/30350). I use Fedora Linux with AMD 7600xt GPU.

**Reply by Kn4ughty** [reply]
I use the 7600 non xt version, so maybe its a problem with that specific GPU?

**Comment by 486c** [comment]
Hello, i can reproduce both original map [[1]](https://osu.ppy.sh/beatmapsets/72324#osu/212095) in the discussion as well as [this map](https://osu.ppy.sh/beatmapsets/318093#osu/708466) [[2]](https://osu.ppy.sh/beatmapsets/318093#osu/708466) at slider at `00:03:799 (3)`

![image](https://github.com/user-attachments/assets/630a2aae-7a9f-4225-b8b9-110531d98dfc)

this seems to happen at certain resolutions. In [[2]](https://osu.ppy.sh/beatmapsets/318093#osu/708466) case its starts appearing only at `2560x1440`
in [[1]](https://osu.ppy.sh/beatmapsets/72324#osu/212095) case its starts appearing only at height close to `1080` so at those resolutions its still present:
```
1860x1080
1920x1082
2200x1080
2310x1080
```

After some bisecting found a commit at which it all started - 6ae82c5e73d27d91ad4d11d381cc59d3044784c6 
Commit below (e1e9c3d7b3) - no issues at both beatmaps

On Vulkan renderer no issues appeared on any version/commit

UPD: Issue still happens at both Legacy and Experimental OpenGL 

Hardware/Software info:
Linux 6.12.7 | Wayland | OpenGL Renderer
GPU: RX 7900XTX
Drivers: mesa-25.0.0

**Comment by velamyGH** [comment]
happening to me as well, also on a 7000 series card which seems to be the consistency. heres a couple maps that it happens on just in case

( 00:51:530 (1) - ) on https://osu.ppy.sh/beatmapsets/2157056#osu/4560237
( 00:47:499 (5) - ) on https://osu.ppy.sh/beatmapsets/2263303#osu/4817853

hw/sw info:

EndeavourOS x86_64 | Linux 6.14.6-arch1-1
CPU: AMD Ryzen 5 3600 (12) @ 4.21 GHz
GPU: AMD Radeon RX 7700 XT [Discrete] | mesa 1:25.1.1-1 | Wayland
Renderer: OpenGL
